src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -535,7 +535,8 @@ private static void AddCertificateChainToTheUserCertificateAuthority(X509Certifi
             else
                 collection.Import(rawBytes, password, X509KeyStorageFlags.PersistKeySet | X509KeyStorageFlags.Exportable | X509KeyStorageFlags.MachineKeySet);
 
-            using (var userIntermediateStore = new X509Store(StoreName.CertificateAuthority, StoreLocation.CurrentUser, 
+            var storeName = PlatformDetails.RunningOnMacOsx ? StoreName.My : StoreName.Root;
```

Comment:
accidentally changed StoreName.CertificateAuthority to StoreName.Root?

Reply:
fixed